modules/planning/reference_line/cos_theta_reference_line_smoother.h

```diff
@@ -43,25 +44,31 @@ class CosThetaReferenceLineSmoother : public ReferenceLineSmoother {
   void SetAnchorPoints(const std::vector<AnchorPoint>&) override;
 
  private:
-  bool Smooth(const std::vector<Eigen::Vector2d>& point2d,
-              const std::vector<double>& lateral_bounds,
-              std::vector<common::PathPoint>* ptr_smoothed_point2d);
+  bool CosThetaSmooth(const std::vector<Eigen::Vector2d>& point2d,
```

Comment:
Better rename it to smooth; the function is defined in CosThetaReferenceLineSmoother, no need to specifically state CosThetaSmooth.